cases/interactive-flows.optional.test.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * @jest-environment ./cases/environment.js
+ */
+import { fetch } from "./util/fetchShim";
+import getSep10Token from "./util/sep10";
+import StellarSDK from "stellar-sdk";
+import getTomlFile from "./util/getTomlFile";
+import { getTransactionBy } from "./util/transactions";
+import { doInteractiveFlow } from "./util/interactive";
+import { getTransactionSchema } from "./util/schema";
+const urlBuilder = new URL(process.env.DOMAIN);
+const url = urlBuilder.toString();
+const keyPair = StellarSDK.Keypair.random();
+const horizonURL = "https://horizon-testnet.stellar.org";
+const server = new StellarSDK.Server(horizonURL);
+
+jest.setTimeout(180000);
+
+let infoJSON;
+let enabledCurrency;
+let jwt;
+let toml;
+let currencyInfo;
+let issuerAccount;
+let asset;
+let account;
+
+function waitUntilTruthy(valObj, timeAllowed, interval) {
+  return new Promise(async (resolve, reject) => {
+    await new Promise(async (resolve) => {
+      let timePassed = 0;
+      while (!valObj.val && timePassed < timeAllowed) {
+        await new Promise((resolve) => {
```

Comment:
for clarity, can we move these sleep promises out into a sleep function?  `await sleep(interval)` seems more readable